src/en/upgrade-series.md

```diff
@@ -0,0 +1,124 @@
+Title: Updating a machine series
+TODO:  warning: ubuntu code names hardcoded
+
+# Updating a machine series
+
+Starting with Juju `v.2.5.0`, to upgrade the series of a machine, the
+`upgrade-series` command is used.
+
+!!! Important:
+    The `upgrade-series` command does not support upgrading a controller. See
+    below section [Upgrading a controller machine][#upgrading-a-controller] for
+    how to do that using an alternate method.
+
+## Upgrading a workload machine
+
+Here is an overview of the process:
+
+ 1. The user initiates the upgrade.
+    1. The machine is removed from the pool of available machines in the sense
+       that charms can not be deployed upon it.
+    1. A minimum amount of the machine's software is upgraded and some changes 
+       internal to Juju are made. 
+    1. All units on the machine are taken into account.
+    1. At the end of this step, from Juju's standpoint, the machine is now
+       running the target operating system.
+ 1. The user manually performs the upgrade of the operating system and makes
+    any other necessary changes. This should be accompanied by a maintenance
+    window managed by the user.
+ 1. The user informs Juju that the machine has been successfully upgraded. The
```

Comment:
I was using "pool" colloquially. I will remove as I agree it can be confusing. There is the notion of "available machines" though. If you `add-machine -n3 ` you have a "pool" of 3 available machines for `deploy` to use. Since a 'prepared' machine is not supposed (bug?) to allow deployments it kind of makes sense. Let me know.

Reply:
I removed "pool" and added "inability to have charms deployed" (bug fix pending).